### PR TITLE
Docs: Ledger

### PR DESCRIPTION
**Pull request by johnyob**
# Context

**Related Tasks**: [API doc: Ledger](https://app.asana.com/0/1205170694555856/1205457666696797/f)

# Description

**Dependencies**:  #36, #35  

This PR an API reference and quick start for the `Ledger` runtime API

# Manually testing the PR

```sh
npm run docs:dev
```

### CODE REVIEW

**Review by alanmarkoTrilitech — APPROVED** [review]
LGTM!

File: docs/api/ledger.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+```typescript
+Ledger.transfer(alice, 420); // Transfer 420 mutez to Alice from the balance of the smart function
+console.log(Ledger.balance(alice)); // 420
+console.log(Ledger.balance(Ledger.selfAddress)); // 0
+```
+
+## Instance Properties
+
+### `readonly Ledger.selfAddress: Address`
```
Comment (asutosh-mourya) [inline docs/api/ledger.md:34]: Where will the definition of type Address be available?

File: docs/api/ledger.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+## Instance Methods
+
+### `Ledger.balance(address: Address): Mutez`
+
+Returns the balance of the given address in mutez, or `0` if the address is not in the ledger.
+
+### `Ledger.transfer(dst: Address, amount: Mutez): void`
+
+Transfers the given amount of mutez from the balance of the smart function to the given address. If the smart function does not have enough balance, this throws an error.
```
Comment (asutosh-mourya) [inline docs/api/ledger.md:46]: Should we think of defining types of errors in future for easier handling?

File: docs/api/ledger.md (hunk 1 above)
Comment (johnyob) [reply docs/api/ledger.md:34]: I will add `Address` to the `ledger.md` docs

File: docs/api/ledger.md (hunk 2 above)
Comment (johnyob) [reply docs/api/ledger.md:46]: The errors raised are `JsNativeError`s which may be had to define in TS. Alternatively, the SDK could wrap over these functions and define non-native errors. 

I will discuss this further with the Boa team.

**Review by sam-finch-tezos — APPROVED** [review]

**Pull request merged**